skyvern/cli/run_commands.py

```
@@ -163,3 +167,135 @@ def run_mcp() -> None:
     # This breaks the MCP processing because it expects json output only
     # console.print(Panel("[bold green]Starting MCP Server...[/bold green]", border_style="green"))
     mcp.run(transport="stdio")
+
+
+@run_app.command(
+    name="code",
+    context_settings={"allow_interspersed_args": False},
+)
+def run_code(
+    script_path: str = typer.Argument(..., help="Path to the Python script to run"),
+    params: List[str] = typer.Option([], "-p", help="Parameters in format param=value (without leading dash)"),
```

Ruff (0.14.0)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)